certification/internal/engine/engine.go

```diff
@@ -100,7 +100,15 @@ func (c *CraneEngine) ExecuteChecks() error {
 		return fmt.Errorf("%w: %s", errors.ErrExtractingTarball, err)
 	}
 
-	if err := writeCertImage(img); err != nil {
+	// store the image internals in the engine image reference to pass to validations.
+	// TODO: pass this to validations when the new check interface includes it.
```

Comment:
This TODO can be removed

Reply:
I've been wondering that for awhile and keep forgetting to ask, will remove it.